src/vector_tile_geometry_decoder.ipp

```diff
@@ -248,61 +272,79 @@ void decode_linestring(mapnik::geometry::geometry<typename T::value_type> & geom
         {
             geom = std::move(*itr);
         }
+        else
+        {
+            geom = std::move(mapnik::geometry::geometry_empty());
```

Comment:
@flippmoke:

1. What is the reasoning for additional `else` in general? E.g. why set to empty vs what was it before?
2. I think the `std::move` is useless here since `mapnik::geometry::geometry_empty()` is already a temporary and it can be just `geom = mapnik::geometry::geometry_empty();` /cc @artemp for confirmation.